Automattic/pocket-casts-ios
# Fix keyboard shortcuts order for MainTabBarController

Fixes #1951 

Keyboard shortcuts were not updated with the addition of Up Next to tab bar (see #1613). This PR adds a keyboard shortcut for the Up Next tab, and corrects the shortcut for the Profile tab.

## To test

1. Run on iPad
2. Ensure that `upNextOnTabBar` feature flag is enabled
3. Hold Cmd on keyboard (works fine in sim). Ensure that listed keyboard shortcuts for Tab Bar (Cmd+1, Cmd+2, etc) lines up with the actual order on the tab bar
4. Test each shortcut (Cmd+1 through Cmd+5) and ensure that navigation goes to the correct tab.

![image](https://github.com/user-attachments/assets/9ccba881-b142-4800-b900-abe89c364e42)

5. Disable `upNextOnTabBar` feature flag.
6. Kill and restart the app
7. Up Next should no longer be on tab bar. Hold Cmd and ensure that shortcut for Up Next is also gone.
8. Test each shortcut (Cmd+1 through Cmd+4) and ensure that navigation goes to the correct tab.

## Checklist
